checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -426,39 +428,44 @@ private void trackInvocationResult(Set<Obligation> obligations, Node node) {
       return;
     }
 
-    // `mustCallAlias` is the argument passed in the MustCallAlias position, if any exists,
-    // otherwise null.
-    Node mustCallAlias = getMustCallAliasArgumentNode(node);
-    // If mustCallAlias is null and call returns @This, set mustCallAlias to the receiver.
-    if (mustCallAlias == null
+    // `mustCallAliases` is a set of the argument(s) passed in the MustCallAlias position(s), if any
```

Comment:
You can just write, `// mustCallAliases is a (possibly-empty) list of arguments passed in a MustCallAlias position`